sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationClientImpl.java

```diff
@@ -0,0 +1,796 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
This looks like an autorest generated class but I don't see the codegen comment here. So, I am curious if this was manually added or generated and for some reason, the comment is missing.

Reply:
Yes, this was handwritten.